src/plugin-api.js

```diff
@@ -126,6 +126,18 @@ export default class PluginAPI {
       config.app.type = 'meteor';
     }
 
+    if (utils.configHasMailUrl(config)) {
```

Comment:
Please move these to `prepareConfig` in the Meteor plugin.

Reply:
Done.